gateway/src/main/java/io/zeebe/gateway/impl/configuration/ClusterCfg.java

```diff
@@ -30,10 +32,10 @@
 import java.util.Objects;
 
 public class ClusterCfg {
-
   private String contactPoint = DEFAULT_CONTACT_POINT_HOST + ":" + DEFAULT_CONTACT_POINT_PORT;
   private String transportBuffer = DEFAULT_TRANSPORT_BUFFER_SIZE;
   private String requestTimeout = DEFAULT_REQUEST_TIMEOUT;
+  private String clusterName = DEFAULT_CLUSTER_NAME;
```

Comment:
New configuration property is not tested nor documented in the distribution configuration file